packages/yourloops/components/menus/user-menu.tsx

```diff
@@ -125,68 +127,71 @@ function UserMenu(): JSX.Element {
   }, [userName])
 
   return (
-    <React.Fragment>
-      <Box
-        id="user-menu"
-        display="flex"
-        alignItems="center"
-        className={clickableMenu}
-        maxWidth={250}
-        onClick={event => {
-          setAnchorEl(event.currentTarget)
-        }}
-      >
-        <Box display="flex">
-          {getRoleIcon()}
-        </Box>
-        {!isMobileBreakpoint &&
-          <React.Fragment>
+    <>
+      <Box maxWidth={250}>
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏Fixing the width here makes the text ellipsis not  working anymore. Should be setted on the Typography line 148 😉

Reply:
Should be fixed now